### ISSUE TITLE
oh-my-zsh.sh:7: parse error near `<<<'

### ISSUE THREAD

**Issue by f0kuz** [ohmyzsh/ohmyzsh#10281]
Hi, how to fix it? This was occur after upgrade, now it looks like that:

```
# Set ZSH_CACHE_DIR to the path where cache files should be created
# or else we will use the default cache/
if [[ -z "$ZSH_CACHE_DIR" ]]; then
  ZSH_CACHE_DIR="$ZSH/cache"
fi

<<<<<<< Updated upstream
# Make sure $ZSH_CACHE_DIR is writable, otherwise use a directory in $HOME
if [[ ! -w "$ZSH_CACHE_DIR" ]]; then
  ZSH_CACHE_DIR="${XDG_CACHE_HOME:-$HOME/.cache}/oh-my-zsh"
fi

# Create cache and completions dir and add to $fpath
mkdir -p "$ZSH_CACHE_DIR/completions"
(( ${fpath[(Ie)"$ZSH_CACHE_DIR/completions"]} )) || fpath=("$ZSH_CACHE_DIR/completions" $fpath)

=======
# Migrate .zsh-update file to $ZSH_CACHE_DIR
if [ -f ~/.zsh-update ] && [ ! -f ${ZSH_CACHE_DIR}/.zsh-update ]; then
    mv ~/.zsh-update ${ZSH_CACHE_DIR}/.zsh-update
fi

>>>>>>> Stashed changes
# Check for updates on initial load...
if [ "$DISABLE_AUTO_UPDATE" != "true" ]; then
  env ZSH=$ZSH ZSH_CACHE_DIR=$ZSH_CACHE_DIR DISABLE_UPDATE_PROMPT=$DISABLE_UPDATE_PROMPT zsh -f $ZSH/to$
fi
```

**Comment by esturniolo** [comment]
Oh... I see... So the solution should be reinstall it? Unintall it and install it again?

Thanks.

**Comment by rwmitchell** [comment]
You should look to see what you changed before reinstalling. In the future, make that change in your .zshrc file. (probably)

**Comment by esturniolo** [comment]
I don't deny changes, because I don't remember to make changes.... I made some changes but in the .zshrc file. 

In the worst case that I decide to uninstall and reainstall it, the only file that I should backup shoul be .zshrc, rigth?

**Comment by carlosala** [comment]
You can execute `cd $ZSH; git reset .; omz update` and you should be ready to go.

**Comment by esturniolo** [comment]
> You can execute `cd $ZSH; git reset .; omz update` and you should be ready to go.

`Hooray! Oh My Zsh has been updated!`

That owrked, but I found another problem in my zsh... When I open a terminal starts a zsh session. I tried to execute `zsh` again and surprise! The Oh-My-Zsh (includig powerlevel10k theme) starts without any problem.

So it's like the default zsh session, doesn't recognize nothig by default.

Ideas?

**Comment by rwmitchell** [comment]
Yes.  You wiped out the changes you made (in the wrong place) when you did 'git reset'.

But, since you looked at the changes before wiping them out, use that information to put them in your .zshrc file

**Comment by esturniolo** [comment]
Hi @rwmitchell 
Nope. But I saw the difference and it was some `if / elseif` in the zsh theme only.

I created a `.zprofile` file where I put `zsh` in the begging. So now I've the zsh properly working. Same with OMZ.
It's working now.

But now my Warp console is broken.... I fix something and then another thing brakes.

**Comment by nohren** [comment]
This is a merge conflict due to editing the file in the wrong place.

https://github.com/ohmyzsh/ohmyzsh/wiki/Customization

For editing the theme.

TLDR don't edit themes in the themes directory. Copy paste the them you want to edit into the custom/theme directory. These take precedence over themes.
